examples/ingest-hn/main.go

```diff
@@ -0,0 +1,111 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+	"sync"
+
+	"github.com/axiomhq/axiom-go/axiom"
+)
+
+const BaseURL = "https://hacker-news.firebaseio.com"
+
+func getMaxItemID() (uint64, error) {
+	res, err := http.Get(BaseURL + "/v0/maxitem.json")
+	if err != nil {
+		return 0, fmt.Errorf("failed to get maxitem.json: %w", err)
+	}
+	defer res.Body.Close()
+
+	maxItemIDBytes, err := ioutil.ReadAll(res.Body)
```

Comment:
```suggestion
	maxItemIDBytes, err := io.ReadAll(res.Body)
```

`ioutil` is deprecated.